packages/yoga/src/Tag/native/Informative.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import styled, { withTheme } from 'styled-components';
+import { func, oneOfType, oneOf, bool, node } from 'prop-types';
+import { hexToRgb } from '@gympass/yoga-common';
+import { StyledTag, StyledText } from './Tag';
+
+const Informative = styled(StyledTag)`
+  ${({
+    variant,
+    theme: {
+      yoga: {
+        colors: {
+          text,
+          feedback: {
+            [variant]: { light = text.secondary },
+          },
+        },
+        components: { tag },
+      },
+    },
+  }) => `
+    background-color: ${hexToRgb(light)};
+    border-color: ${light};
+    border-radius: ${tag.border.radius}px;
+  `}
+`;
+
+const Wrapper = styled.View`
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+`;
+
+const StyledTextInformative = styled(StyledText)`
+  ${({
+    theme: {
+      yoga: {
+        colors: { text, color = text.primary },
+        components: { tag },
+      },
+    },
+  }) => `
+    color: ${color};
+
+    font-size: ${tag.font.size}px;
+    line-height: ${tag.font.lineHeight}px;
+    font-weight: ${tag.font.weight};
+  `}
+`;
+
+const TagInformative = ({
```

Comment:
We must have a description for this component. 
You can easily find it on its Figma file, in the `Definition` section.